src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php

```diff
@@ -20,15 +20,13 @@
 use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;
 use Symfony\Component\Routing\Loader\PhpFileLoader as RoutingPhpFileLoader;
 use Symfony\Component\Routing\RouteCollection;
-use Symfony\Component\Routing\RouteCollectionBuilder;
 
 /**
  * A Kernel that provides configuration hooks.
  *
  * @author Ryan Weaver <ryan@knpuniversity.com>
  * @author Fabien Potencier <fabien@symfony.com>
  *
- * @method void configureRoutes(RoutingConfigurator $routes)
  * @method void configureContainer(ContainerConfigurator $container)
```

Comment:
Didn't you forget this one?

Reply:
we also accept this (and did not deprecate it to not force using the DSL for the sake of it):
`configureContainer(ContainerBuilder, LoaderInterface)`